fuzz/fuzz.ml

```diff
@@ -3,16 +3,16 @@
 
 open Crowbar
 
-exception Deflate_error of Decompress.Zlib_deflate.error
-exception Inflate_error of Decompress.Zlib_inflate.error
+exception Deflate_error of Decompress.Gzip_deflate.error
```

Comment:
Sorry for that, i totally forgot the fuzzing thing, i'm doing my first steps in the community and i'm not used to these tools.
I'm not even sure it would work with the actual gzip implementation.
I will look at it as soon as i can

Reply:
No worries, I appreciate the effort you put into this.
Please don't take any of my comments as expressions of negativity, I just wanted to provide constructive criticism, and am happy to see work on a GZip implementation!

Reply:
Criticism is welcome, I'm still a learning student, the more I receive, the more I progress :D